doc/using/configuration.chapter.md

```diff
@@ -200,7 +200,7 @@ There are several ways to tweak how Nix handles a package for which there are wa
     }
     ```
 
-    Note that `ignoreWarningsPackages` is only checked if `ignoreWarningsPredicate` is not specified.
+    Note that `ignoreWarningsPackages` is only checked if `ignoreWarningsPredicate` is not specified. You can manually integrate the expression `builtins.elem (lib.getName pkg) ignoreWarningsPackages` into the predicate to still make use of the package list in addition to the custom predicates.
```

Comment:
Always be explicit and provide full examples. We can make it easier for readers to make use of our instructions.

````suggestion
    Note that `ignoreWarningsPackages` is only used if `ignoreWarningsPredicate` is not specified.
    
    You can manually inspect the list `ignoreWarningsPackages` to combine it with custom predicates, as in the following example:
    
    ```nix
    {
      ignoreWarningsPredicate = pkg: 
        builtins.elem (lib.getName pkg) ignoreWarningsPackages
        ||
        lib.all (issue: issue.kind == "deprecated") (pkg.meta.issues or [])
    }
    ```
````

This is probably a weird way of formatting it. Feel free to come up with something that fits to extending the previous example and is still readable.

Reply:
I agree it's not ideal. Before applying your suggestion though, I'll try to come up with a solution where users don't have to deal with such thing in the first place.